src/org/rascalmpl/library/List.rsc

```diff
@@ -480,9 +490,10 @@ list[&T] push(&T elem, list[&T] lst) = [elem] + lst;
 
 
 @synopsis{Apply a function to successive elements of list and combine the results.}
-@deprecated{This function is deprecated. Use a reducer expression instead, like `(init | f(it, e) | e <- lst)`.}
```

Comment:
why is it undeprecated? (in a PR that's about cleaning up deprecated API)

Reply:
@PaulKlint  had asked for a motivation why this is deprecated. The only reason is because it is superfluous. But apparently there are uses. So instead we document the overlap and let users pick